.expeditor/release.omnibus.yml

```diff
@@ -29,9 +29,9 @@ builder-to-testers-map:
   el-7-x86_64:
     - el-7-x86_64
     - el-8-x86_64
-  freebsd-10-x86_64:
-    - freebsd-10-x86_64
-    - freebsd-11-x86_64
+  freebsd-10-amd64:
+    - freebsd-10-amd64
+    - freebsd-11-amd64
```

Comment:
Is this a dramatic change in architecture we support or only a labeling change?

Reply:
only a labeling change as mentioned in the PR comment. good catch though.